2022/2022-08-29-announcements.md

```diff
@@ -0,0 +1,52 @@
+# CFDE Announcements
+
+## Cross-Pollination Meeting
+Next week our CP Meeting will have a presentation on the CFDE Senescence Network (SenNet). We have space for another talk if you are ready to present! The meeting will take place on **Tuesday, Septeber 6th, at 11am PT / 2pm ET**.  Our remaining 2022 Cross-Pollination meetings will be showcasing CFDE Partnership Projects as reflected on the schedule. However, there is still space for additional talks each month. If you are interested please sign up to present at an upcoming Cross Pollination Meeting. Regular Cross-Pollination events take place on the first Tuesday of every month (Sep 6, Oct 4, Nov 8, and Dec 6), and we have 7 slots available. View the [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) for more details.  Email [support@cfde.atlassian.net](mailto:support@cfde.atlassian.net) if you would like to present.  
+
+## Content Registry Release
+We are pleased to announce that the content registry is now up on the portal!! Thanks to participation from Glycosciences, IDG, LINCS, Metabolomics Workbench, and SPARC, this first release annotates the following resources in the portal:
+
+- anatomy: 321 of 334
+- compound: 45898 of 73498
+- disease: 1872 of 1894
+- gene: 19962 of 19975
+- protein: 64145 of 64145
+
+Here are a few links to explore and share:
+- GTEx expression data on an anatomy page:
+https://app.nih-cfde.org/chaise/record/#1/CFDE:anatomy/id=UBERON%3A0000007
+- Compound information and visualization:
+https://app.nih-cfde.org/chaise/record/#1/CFDE:compound/id=G99997WS
+- Links to the knowledge graph on compound page:
+https://app.nih-cfde.org/chaise/record/#1/CFDE:compound/id=10000456
+- MANY links and visualization and Appyters on a gene page:
+https://app.nih-cfde.org/chaise/record/#1/CFDE:gene/id=ENSG00000000003
+- Integration of genes with Metabolomics and LINCS on a gene page:
+https://app.nih-cfde.org/chaise/record/#1/CFDE:gene/id=ENSG00000000419
+
```

Comment:
Add "View the release notes for more information about the workflows and contributors https://github.com/nih-cfde/update-content-registry/releases/tag/2022.08." as the final paragraph